webcompat/helpers.py

```diff
@@ -100,6 +100,18 @@ def get_user_info():
         session['avatar_url'] = gh_user.get('avatar_url')
 
 
+def version_string(json_section):
```

Comment:
Can we rename the method to `get_version_string`?

Also, the argument being called `json_section` is a bit hard to understand what exactly it is. It should be a Python Dict, not a json object, right? Maybe we can rename to something like `user_agent_dict` (or just `user_agent`, with a comment that the arg is a dict, in the docstring)?